backend/src/controllers/rentals.py

```diff
@@ -10,31 +10,38 @@
 from ..models.task import Task
 from ..models.vehicle import Vehicle
 from ..models.insurance import Insurance
+from ..models.feedback import Feedback
 import base64
 
-rentals = Blueprint('rentals', __name__, url_prefix='/rentals')
 rentals = Blueprint('rentals', __name__, url_prefix='/rentals')
 response_class = Blueprint('response_class', __name__)
 
 
+def check_if_feedback_exist(rental):
+    if Feedback.query.filter_by(rental_id=rental.rental_id).first():
+        return True
+    else:
+        return False
```

Comment:
Nie wiem czy dobrze rozumiem, ale czy można to do returna odrazu wrzucić ?

Reply:
można zrobić return  Feedback.query.filter_by(rental_id=rental.rental_id).first() is not None